swagger/en/schemas/gameModeStats.yml

```diff
@@ -34,7 +42,12 @@ properties:
     type: integer
     description: "Number of healing items used"
     minimum: 0
-  killPoints:
+  killPoints [PC]:
+    type: number
+    description: "Points based on how many kills this player had"
+    minimum: 0
+    deprecated: true
+  killPoints [Xbox]:

```

Comment:
After looking at this more, I'm not sure it will be okay. People might expect these field names will actually have ` [PC]` and ` [Xbox]` in them in the real data. Maybe we could just leave a note in the description saying "deprecated for PC" or something?